### ISSUE TITLE
RightToLeft's FlowDirection not working when Force to ContentPage

### ISSUE THREAD

**Issue by jrahma** [xamarin/Xamarin.Forms#3501]
Hi,

I have tried to set the ContentPage's FlowDirection to RightToLeft and it's working beautifuly on my Android device when I change the device language to Arabic

but when I change the language back to English then few things won't work, as an  example Header and back button directions although the FlowDirection is set to RightToLeft not to Device.FlowDirection

**Comment by samhouts** [comment]
@jrahma Do you have a small reproduction project you can share? Thank you!

**Comment by jrahma** [comment]
Try the attached... It works when the device language is set to Arabic but when you change it to English then it will move the header to LTR although the FlowDirection is set to RightToLeft in the **News** page  and the android:supportsRtl="true" and the MinSDKVersion is 17+. as per your [article](https://blog.xamarin.com/right-to-left-localization-xamarin-forms/)

please see the attached video..


[RightToLeft.zip](https://github.com/xamarin/Xamarin.Forms/files/2273605/RightToLeft.zip)

[RightToLeft.zip](https://github.com/xamarin/Xamarin.Forms/files/2273615/RightToLeft.zip)

**Comment by samhouts** [comment]
Yes, there are a few things that we were unable to force the flow direction on during our first pass. The Navigation Bar is one of them, and more are listed in the Known Issues of #1222. We will work on improving this support. Thank you!

**Comment by SagarPanwala** [comment]
FlowDirection applied in parent is also not working on ListView HeaderTemplate

**Comment by jrahma** [comment]
@samhouts 

This is still not fixed?

Please see below screenshot

The page is set to RightToLeft and everything is fine except the Page Title

![Screenshot_20190805_062745_com jafaria official](https://user-images.githubusercontent.com/6236241/62434675-4edbc780-b74a-11e9-8198-d15fbde1b46b.jpg)


Thanks,
[Jassim](https://www.softnames.com)

**Comment by samhouts** [comment]
@jrahma This is on the backlog. I do not currently have an estimate of when we may begin working on it. Thanks for your patience!

**Comment by nikhildpardasani** [comment]
Even I am facing the same issue and we have a release of the App in near future. When can we have a this issue fixed. ?
Kindly work on this on priority.

**Comment by KevinAnass** [comment]
i have the same issue in ios

**Comment by ameralfaqeeh** [comment]
### Add the following to android.manifest 
                   android:supportsRtl="true"
### Like the following:
```
  <application android:label="app2.Android" android:allowBackup="false" android:supportsRtl="true"                        android:hardwareAccelerated="true" android:theme="@style/MainTheme">
  </application>
```

### I hope this can help

**Comment by ahmeduday** [comment]
I have similar issue, Solved by:
1. put FlowDirection="{x:Static Device.FlowDirection}" in a parent of the listview (may be also possible in the listview,, Ididn't try)
2. put FlowDirection="MatchParent" in the listview.HeaderTeplate main layout e.g. <Grid FlowDirection="MatchParent">
